app/mailers/notification_mailer.rb

```diff
@@ -0,0 +1,6 @@
+class NotificationMailer < ApplicationMailer
+  def cards_for_review_available(user)
+    @user = user
+    mail(to: @user.email, subject: 'Cards for review available')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.